webextensions/browser-compat-data.json

```diff
@@ -692,10 +692,10 @@
     }, 
     "sendNativeMessage": {
       "Chrome": {
-        "support": true
+        "support": "26"
```

Comment:
"28" according to the Chrome docs: https://developer.chrome.com/extensions/runtime#method-sendNativeMessage